src/ratbag_emu/protocol/steelseries.py

```diff
@@ -34,10 +34,10 @@ class SteelseriesDevice(BaseDevice):
     '''
     Init routine
     '''
-    def __init__(self, rdesc=None, info=None,name='Generic Steelseries Device',
-                 shortname='generic-steelseries', protocol_version=2):
-        self.protocol = f'Steelseries v{protocol_version}'
-        super().__init__(rdesc, info, name, shortname)
+    def __init__(self, obj):
+        super().__init__(obj)
+
+        protocol_version = obj['steelseries']['version']
```

Comment:
Again, there should not be any point in modifying steelseries.
It should be converted in a separate commit, when/if there is a need to modify it.

Reply:
But these modifications come from a change in the API, this would introduce a breaking commit. I agree this commit is a mess but I thought this specific change was correct, I can change it though.